backend/danswer/dynamic_configs/store.py

```diff
@@ -82,19 +46,36 @@ def store(self, key: str, val: JSON_ro, encrypt: bool = False) -> None:
             session.commit()
 
     def load(self, key: str) -> JSON_ro:
+        redis_value = self.redis_client.get(key)
+        if redis_value:
+            if isinstance(redis_value, bytes):
+                return json.loads(redis_value.decode("utf-8"))
```

Comment:
I imagine utf-8 should cover everything we need? Any concerns here?

Reply:
Looks OK to me.